documentation/src/main/asciidoc/topics/json/persistence_sql_query.json

```diff
@@ -6,24 +6,18 @@
         "shared": "true",
         "key-columns": "isbn",
         "queries": {
-          "select-single": "SELECT isbn, title FROM books WHERE isbn = :isbn",
-          "select-all": "SELECT isbn, title FROM books",
+          "select-single": "SELECT books.id, books.isbn, title, authors.name, authors.country FROM books JOIN authors ON books.author_id = authors.id WHERE books.id=:id",
+          "select-all": "SELECT books.id, books.isbn, title, authors.name, authors.country FROM books JOIN authors ON books.author_id = authors.id",
           "delete-single": "DELETE FROM books WHERE isbn = :key",
           "delete-all": "DELETE FROM books",
           "upsert": "INSERT INTO books (isbn, title) VALUES (:key, :value) ON CONFLICT (ispn) DO UPDATE SET title = :value",
-          "size": "SELECT COUNT(*) FROM books"
+          "size": "SELECT COUNT(*) FROM books JOIN authors ON books.author_id = authors.id"
```

Comment:
No need for the join as the books table is the one that has all the values, since it is N books per 1 author.